### PR TITLE
Updated Codeship

### PR DESCRIPTION
**Pull request by fbernhart**
codeship

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

**Comment by adamrusted** [comment]
Looks like this one is contained within the circle when it's displayed on the site. Should we look to maintain the circle where possible?
Also looks like some of the paths have been smoothed out in the PR version.

image

**Comment by fbernhart** [comment]
@adamrusted Thanks for reviewing!

I agree, that the circle should be added. In my opion as well in a "design perspective". Otherwise the rounded shape of the waves wouldn't really make sense, if they weren't bordered by the circle.


I've as well updated the color, using the darker blue of the ship `#004466`.
codeship

File: icons/codeship.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Codeship icon</title><path d="M12 0C5.372 0 0 5.372 0 12s5.372 12 12 12 12-5.372 12-12S18.628 0 12 0zm0 1.334c.824 0 1.636.191 2.373.56a.53.53 0 0 1 .294.474v1.519a15.985 15.985 0 0 0-5.334 0V2.368c0-.2.114-.384.294-.474A5.304 5.304 0 0 1 12 1.334zm5.143 4.595c.114.043.19.152.19.274v2.122A20.956 20.956 0 0 0 12 6.648c-1.951.344-3.794.943-5.333 1.677V6.203c0-.122.076-.231.19-.274a14.648 14.648 0 0 1 5.038-.933c1.926-.024 3.725.37 5.248.933zM12 8s3.752.625 6.411 2.482c.145.101.18.299.084.448-1.104 1.74-1.97 3.922-2.596 5.838 1.51-1.579 2.473-2.068 4.1-2.101.9 0 1.598.202 2.185.516A10.637 10.637 0 0 1 12 22.667c-4.32.044-8.341-2.684-9.914-6.723.8-.709 1.74-1.277 3.247-1.277.92 0 1.626.212 2.22.537-.554-1.475-1.236-2.994-2.048-4.274a.33.33 0 0 1 .084-.448C8.248 8.625 12 8 12 8zm0 1.333c-1.333 2 0 8 0 8s1.333-6 0-8z"/></svg>
```
Comment (adamrusted) [inline icons/codeship.svg:1]: There are some ***very*** minor differences between your PR (red) and my from-source (black), which could just be a result of optimizing areas with a lot of additional points. I'm happy with this, if another @simple-icons/maintainers is happy to sign off on it too?

image

**Review by adamrusted — APPROVED** [review]

File: icons/codeship.svg (hunk 1 above)
Comment (fbernhart) [reply icons/codeship.svg]: @adamrusted Is it better now? I've now slightly corrected it.

It happened when I tried to create a difference between the ship and the circle. Inkscape then created around 100 additional points (don't ask me why). So I deleted them and manually tried to fit the curve as much as possible to the original SVG. 😄

**Review by adamrusted — APPROVED** [review]
LGTM! 🎉

**Pull request merged**